nucleus/transform.py

```diff
@@ -0,0 +1,275 @@
+import numpy as np
+import transforms3d as t3d
```

Comment:
2 more undeclared dependency that needs to be added so when users

pip install scale-nucleus

they won't get an error when running this file.